src/transformers/models/auto/configuration_auto.py

```diff
@@ -411,8 +411,8 @@ def __contains__(self, item):
 
 def _get_class_name(model_class: Union[str, List[str]]):
     if isinstance(model_class, (list, tuple)):
-        return " or ".join([f":class:`~transformers.{c}`" for c in model_class if c is not None])
-    return f":class:`~transformers.{model_class}`"
+        return " or ".join([f"[`~transformers.{c}`]" for c in model_class if c is not None])
+    return f"[`~transformers.{model_class}`]"
```

Comment:
```suggestion
        return " or ".join([f"[`{c}`]" for c in model_class if c is not None])
    return f"[`model_class`]"
```
Nice catch! No need for the ~transformers. since all those objects are in the init.

Reply:
missed `{}` when fixing above, so restored it back.